docs/Options/GenerateOptions.md

```diff
@@ -3,6 +3,8 @@ Options are optional and case-insensitive
 
 * `--violate` 
     * generate data which violates profile constraints. Options are: `true` or `false` (default)
+* `--dont-violate` <epistemic constraints...>
```

Comment:
Can you link to the "Selective violation" document here

Reply:
Can we use `--dont-violate` multiple times or can it take multiple constraint names?

Reply:
updated